modules/core/src/shaderlib/project/project.glsl.js

```diff
@@ -182,4 +205,13 @@ float project_pixel_size(float pixels) {
 vec2 project_pixel_size(vec2 pixels) {
   return pixels / project_uScale;
 }
+
+// Get rotation matrix that aligns the z axis with the given up vector
+mat3 project_get_orientation_matrix(vec3 up) {
+  vec3 nz = normalize(up);
```

Comment:
Nit: Any chance to get variables names with words instead of letters? 
- Add comment hinting about how this is matrix is calculated, perhaps a link. 
-  I assume `nx` etc are columns or rows? How those form a rotation?
- If one is not actively working with linear algebra this can be  non-obvious.